test/embind/embind.test.js

```diff
@@ -1451,7 +1445,7 @@ module({
             c.delete();
         });
 
-        if (testSettings['ASSERTIONS']) {
+        if (cm['getCompilerSetting']('ASSERTIONS')) {
```

Comment:
Does `cm.getCompilerSetting(...)` not work here?

Reply:
Not with closure.

Reply:
Oh, that was true in my manual testing but not in this test. You are right, fixed.

It seems that this test does not run closure on the code you commented on here, so it works (it's exported from closured code, and used from non-closured code).